loleaflet/src/layer/tile/CanvasTileLayer.js

```diff
@@ -324,11 +324,11 @@ L.CanvasTilePainter = L.Class.extend({
 
 					var key = coords.key();
 					var tile = this._layer._tiles[key];
-//					var invalid = tile && tile._invalidCount && tile._invalidCount > 0;
+					//var invalid = tile && tile._invalidCount && tile._invalidCount > 0;
 					if (tile && tile.loaded) {
 						this.paint(tile, ctx);
 					}
-/*					else
+					/*					else
```

Comment:
This doesn't look right.

Reply:
it should be like 

`/*else `

right?

Reply:
Right. Thanks!